src/cljs/nr/gameboard/utils.cljs

```diff
@@ -0,0 +1,12 @@
+(ns nr.gameboard.utils
+  (:require [nr.gameboard.actions :refer [send-command]]))
+
+(defn inc-dec-overlay
```

Comment:
I'm not a fan of this function name. If `controls` is too general, this is too specific and obscure. Maybe a happy medium could be `stat-controls` or `stat-buttons` or something.

Reply:
Additionally, any particular reason for moving this to a new file instead of keeping it with the other stats functions? Importing it from `nr.gameboard.stats` in `nr.gameboard.main` would work just as well. I'm generally not a fan of single-function namespaces.

Reply:
+1 for `stat-controls`.

I've moved it to `utils` because it's not just used in the `player-stats` module, and we could move other "util-like" functions there, too. But this probably only makes sense if we split up `main.cljs` further, and we can still move things around once we do that. I will put it in `player-stats` for now, keeping things simple.